vendor/modules.txt

```diff
@@ -11,6 +11,9 @@ cloud.google.com/go/logging/internal
 # code.cloudfoundry.org/clock v1.0.0
 ## explicit
 code.cloudfoundry.org/clock
+# github.com/AdaLogics/go-fuzz-headers v0.0.0-20221118232415-3345c89a7c72
+## explicit; go 1.18
+github.com/AdaLogics/go-fuzz-headers
```

Comment:
Looks like the vendor was updated, but the `vendor.mod` (which we currently use) wasn't.